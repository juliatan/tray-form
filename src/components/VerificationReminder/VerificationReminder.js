import React from 'react';
import Icon from './Icon/Icon';
import classes from './VerificationReminder.module.scss';

const verificationReminder = () => (
  <div
    className={classes.VerificationReminder}
    data-testid="verificationReminder"
  >
    <Icon />
    <p>
      Please verify your email address. You should have received an email from
      us already!
    </p>
  </div>
);

export default verificationReminder;
