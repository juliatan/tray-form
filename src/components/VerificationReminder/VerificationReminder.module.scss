@import '../../assets/stylesheets/breakpoints';

.VerificationReminder {
  width: 100%;
  text-align: center;
}

@media (max-width: $mobile) {
  .VerificationReminder {
    font-size: 0.875rem;
  }
}
