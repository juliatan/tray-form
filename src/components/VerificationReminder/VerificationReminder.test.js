import React from 'react';
import { render } from '@testing-library/react';
import VerificationReminder from './VerificationReminder';

describe('VerificationReminder component', () => {
  it('should render', () => {
    const { asFragment } = render(<VerificationReminder />);
    expect(asFragment(<VerificationReminder />)).toMatchSnapshot();
  });
});
