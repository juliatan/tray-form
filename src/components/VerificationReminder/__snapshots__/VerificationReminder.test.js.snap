// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerificationReminder component should render 1`] = `
<DocumentFragment>
  <div
    class="VerificationReminder"
    data-testid="verificationReminder"
  >
    <div
      class="Icon"
    >
      <svg>
        envelope.svg
      </svg>
    </div>
    <p>
      Please verify your email address. You should have received an email from us already!
    </p>
  </div>
</DocumentFragment>
`;
